### DISCUSSION TITLE
ecs/basics-ecs/ecs_01_0101

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#618] [Announcements]
# ecs/basics-ecs/ecs_01_0101

这是一段文字。。。。。。。。。 这是一段斜体文字 这是一段加粗文字 ==这是一段高亮文字== HelloWorld $$ y=2x $$ 有序列表 1. 第一级 2. 第二级 1. 第2.1级 无序列表
1111; 
2222; 
33333; 引用框
 这是一个文本框 表格 11111 22222 | ----- ----- | 33333 4444...

http://localhost:8080/ecs/basics-ecs/ecs_01_0101.html

**Comment by cmydream** [comment]
这是一条评论
